docs/.nvmrc

```
@@ -0,0 +1 @@
+v16.16.0
```

Do you think this would work with node.js v18 as well? We are using v18 in the rest of the monorepo so would be handy to be able to use same node installation here too.
```suggestion
v18.16.0
```